### DISCUSSION TITLE
Can i mount OSD PVs in EC2 instance

### DISCUSSION THREAD

**Discussion by ahmedTouati** [rook/rook#8497] [Q&A] [answered]
Our backup contains almost every volume and manifest from rook-ceph cluster, we want to know if we can only mount OSDs Volumes to an EC2 instance as ceph file system and access data in it ?

**Comment by travisn** [comment, marked as answer]
@ahmedTouati No, the data is not directly available by mounting an OSD. Ceph distributes data internally to the OSDs across the cluster. If you want to access the CephFS data, you could mount a RWX volume from multiple pods.

**Reply by ahmedTouati** [reply]
Ok, thanks
